examples/passage_ranker/indexer_reranker_eval_pipeline_test.py

```diff
@@ -0,0 +1,65 @@
+# Copyright 2019 The Forte Authors. All Rights Reserved.
```

Comment:
I realize this is just an example. But for this small feature, there is probably no need to create a new example. If you want you can update an existing one to do this.

Reply:
Updates on this? I feel like there is no need to create a new example that is almost the same as the existing one here: https://github.com/asyml/forte/blob/master/examples/passage_ranker/indexer_reranker_eval_pipeline.py